src/policy/feerate.cpp

```diff
@@ -7,35 +7,31 @@
 
```

Comment:
why are the refactors in 7289e559eb720d972e3e31c0ca81dd8091775b07 needed? They break asan, it seems.

Reply:
The CI was green with the refactors yesterday; the change was the added unit tests. It may be better to propose the refactors in their own PR. I'll drop the refactoring commit to test what the CI says.